Section_11_Reading_And_Writing_Data.ipynb

```diff
@@ -321,7 +321,7 @@
    "source": [
     "### Example - Experimental Data\n",
     "\n",
-    "- Consider an experimental setup whereby air is forced through a rectangular channel. Inside this channel there are several obstacles (cylindrical pins) that obstruct the flow.\n",
```

Comment:
Setup more correct than set-up?

Reply:
I'm not sure, will change it back to "setup".